ml-agents-envs/mlagents_envs/environment.py

```diff
@@ -444,13 +444,15 @@ def close(self):
         else:
             raise UnityEnvironmentException("No Unity environment is loaded.")
 
-    def _close(self):
+    def _close(self, timeout: Optional[int] = None) -> None:
```

Comment:
There is some added logic here, can you add a docstring explaining what the argument does?

Reply:
Done.